test/getusermedia-mocha.js

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
pondering whether I should split that from adapter into a separate repo and pull it via npm...